### PR TITLE
Update common classes to facilitate collaboration among team members

### PR DESCRIPTION
**Pull request by lkldev**
Add methods to model, model manager, tassist and unique student list to help when parsing indexes and student IDs into a list of student objects

Added template for tassist commands that have yet to be created. This will allow those working on the commands to come up with the logic, while at the same time the person doing parsing will be able to create the commands.

### CODE REVIEW

**Review by jxt00 — COMMENTED** [review]
Looks good, thanks for coding it out in such a short time!

File: src/main/java/seedu/address/model/TAssist.java [hunk 1]
```diff
@@ -268,6 +273,25 @@ public String toString() {
...
+        if (indexes.stream().anyMatch(x -> x.getZeroBased() >= getStudentList().size())) {
+            throw new StudentNotFoundException();
+        }
+        return FXCollections.unmodifiableObservableList(
+                FXCollections.observableArrayList(indexes.stream()
+                .map(index -> getStudentList().get(index.getZeroBased()))
+                .collect(Collectors.toList())));
+    }
+
+    public ObservableList<Student> getStudentByIDs(List<StudentId> studentIds) {
```
[the remark this answers is not in the document]
Comment (lkldev) [reply src/main/java/seedu/address/model/TAssist.java]: I did not notice that LOL i change all to getStudentListByStudentIds since we have different Ids

**Review by jxt00 — APPROVED** [review]
Yay LGTM

File: src/main/java/seedu/address/logic/commands/DisenrolCommand.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import seedu.address.model.student.Student;
+
+public class DisenrolCommand extends Command {
+
+    private Index classGroupIndex;
+    private Optional<List<Student>> students;
+    private boolean isAllStudents;
+
+    /**
+     * Creates an EnrollCommand to enroll the students into class group and module
```
Comment (jxt00) [inline src/main/java/seedu/address/logic/commands/DisenrolCommand.java:17]: Should be `Disenrol` but I guess it's ok to let the person in charge fix it.

Comment (lkldev) [reply src/main/java/seedu/address/logic/commands/DisenrolCommand.java:17]: yeap can let the person who editing it to change it

**Review by wxliong — APPROVED** [review]
LGTM! :)

**Pull request merged**